OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -200,5 +200,55 @@ public static Ruleset LoadDefaultsForTileSet(ModData modData, string tileSet)
 
 			return ruleset;
 		}
+
+		static bool AnyCustomYaml(MiniYaml yaml)
```

Comment:
The nodes that are being checked contain yaml overrides that customize the default rules/weapons/etc.